### ISSUE TITLE
Any plan for other network backbones??

### ISSUE THREAD

**Issue by chibai** [Cadene/pretrained-models.pytorch#150]
This is just advice.
In production environment, latency is also very important.
Do you have any plan to train small network, such as mobilenet v1&v2, shufflenet v1&v2 and so on???

**Comment by imirzadeh** [comment]
mobilenet, shufflenet, .... are already implemented and pretrained. they are available via torchvision models: https://pytorch.org/docs/stable/torchvision/models.html
